go.mod

```diff
@@ -23,11 +23,11 @@ require (
 	github.com/ooyala/go-dogstatsd v0.0.0-20140922214459-23f2a1659b02 // indirect
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712 // indirect
 	github.com/satori/go.uuid v1.2.0 // indirect
-	github.com/shopspring/decimal v0.0.0-20171108225254-9ca7f51822d2
+	github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24
 	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726 // indirect
 	github.com/siddontang/go-log v0.0.0-20180807004314-8d05993dda07
-	github.com/siddontang/go-mysql v0.0.0-20180802024903-58848a70cf1a
-	github.com/sirupsen/logrus v1.0.4
+	github.com/siddontang/go-mysql v0.0.0-20200424072754-803944a6e4ea
```

Comment:
We need to audit the changes from the version we pin until this new version as this is a high risk change.

Reply:
Yeah, kind of understood that, 89 commits, 4,656 additions and 18,017 deletions.

https://github.com/siddontang/go-mysql/compare/58848a70cf1a...803944a6e4ea

Reply:
I'll look through that list and pull out the highlights in a comment below.